services/plantService.js

```diff
@@ -0,0 +1,11 @@
+const plantDao = require("../models/plantDao");
+
+const plantsDetails = async (plantId) => {
+  const plantsDetails = await plantDao.plantsDetails(plantId);
+
+  return plantsDetails;
+};
```

Comment:
```suggestion
const getPlantInfo = async (plantId) => {
  return plantDao.getPlantInfo(plantId);
};
```
변수 선언 없이 곧바로 return 해주셔도 됩니다.